src/css/variablesCustom.css

```diff
@@ -225,12 +225,25 @@
   --header-nav-right-background-color-custom: white;
   --header-nav-right-content-width: var(--any-content-width);
   --header-nav-right-logo-width-custom: calc(100rem/18);
-  --header-nav-right-height-mobile-custom: 3.75rem;
+  --header-nav-right-height-mobile-custom: auto;
   --logo-default-height-mobile-custom: var(--header-nav-right-height-mobile-custom);
   --logo-default-width-mobile-custom: var(--logo-default-height-mobile-custom);
   --header-nav-right-height-custom: calc(100rem/18);
   --header-nav-right-margin-custom: 0 0 1rem 0;
   --header-nav-right-padding: 0;
+  --header-nav-grid-a-menu-icon-height: 2px;
+  --header-content-width: calc(min(100%, var(--body-max-width)) - 2.5rem);
+  --header-content-width-mobile: calc(min(100%, var(--body-max-width)) - 1rem);
+  --header-easy-portal-content-width: calc(min(100%, var(--body-max-width)) - 2rem);
+  --header-easy-portal-content-width-mobile: calc(min(100%, var(--body-max-width)) - 1rem);
+  --header-checkout-content-width: calc(min(100%, var(--body-max-width)) - 2rem);
+  --header-checkout-content-width-mobile: calc(min(100%, var(--body-max-width)) - 1rem);
+  --header-border-bottom: 1px solid var(--m-gray-300);
+  --multi-level-navigation-default-o-nav-wrapper-top: 2.7rem;
+  --multi-level-navigation-default-desktop-main-wrapper-width: 105vw;
```

Comment:
Hi all,

We need to use 105vw because, for some reason, 100vw doesn't cover the whole background on Safari, even if it has a position of relative or absolute. Therefore, I decided to add position: absolute and increase the width to be larger than the screen width so that it will always cover the whole side.

The problem is probably with right: 50% and left: 50%, which somehow calculates differently, leaving a small part uncovered.

I hope this is a bit clearer now.